src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/EndpointSecurityTest.java

```diff
@@ -131,7 +130,7 @@ public void should_return_401_when_bearer_token_is_provided_but_no_service_token
 
         given()
             .relaxedHTTPSValidation()
-            .header(authorizationHeadersProvider.getLawFirmAAuthorizationOnly())
+            .header(authorizationHeadersProvider.getCaseworkerAAuthorizationOnly())
             .contentType(MediaType.APPLICATION_JSON_VALUE)
             .baseUri(testUrl)
             .pathParam("task-id", taskId)
```

Comment:
We already have claim above. Should this be cancel? not part of this PR but just noticed.